### DISCUSSION TITLE
Gotrue, adding a new minor provider.

### DISCUSSION THREAD

**Discussion by oiojin831** [supabase/supabase#5200] [Questions] [answered]
Hello,
I have been researching supabase to use it with my next project.
And I need to use a minor provider(kakao) which is only popular in South Korea.
And found out that to use third party provider it has to be included in "goture".
I am not familiar with go and not sure it will be supported by supabase team.
I can try to contribute to the supabase/gotrue but not sure if it is a feasible way to do it.

Thanks for the great product!.

**Comment by GaryAustin1** [comment]
I'd recommend you go contact HarryET#2954 on the Supabase discord channel.  He could probably answer what is involved or point you in the right direction.  He seems to have done several gotrue third party integrations.  I don't know his "handle" here.
There is also a gotrue channel in Supabase there https://discord.com/channels/839993398554656828/932506573987790908.

**Reply by oiojin831** [reply]
@GaryAustin1 Thank you very much :)

**Comment by awalias** [comment, marked as answer]
Hey @oiojin831 thanks for the request! I have added it to the gotrue repo as a requested feature: https://github.com/supabase/gotrue/issues/363

typically these additional providers are added by community members, hopefully someone will be able to take a look 👍

**Reply by oiojin831** [reply]
thank you.
I will try to find out if I can contribute :)
